modules/tensor_mechanics/test/tests/crystal_plasticity/stress_update_material_based/update_euler_angle.i

```diff
@@ -0,0 +1,153 @@
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  elem_type = HEX8
+[]
+
+[AuxVariables]
+  [euler_angle_1]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [euler_angle_2]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [euler_angle_3]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[Modules/TensorMechanics/Master/all]
+  strain = FINITE
+  add_variables = true
+  generate_output = stress_zz
+[]
+
+[AuxKernels]
+  [euler_angle_1]
+    type = MaterialRealVectorValueAux
+    variable = euler_angle_1
+    property = updated_Euler_angle
+    component = 0
+    execute_on = timestep_end
+  []
+  [euler_angle_2]
+    type = MaterialRealVectorValueAux
+    variable = euler_angle_2
+    property = updated_Euler_angle
+    component = 1
+    execute_on = timestep_end
+  []
+  [euler_angle_3]
+    type = MaterialRealVectorValueAux
+    variable = euler_angle_3
+    property = updated_Euler_angle
+    component = 2
+    execute_on = timestep_end
+  []
+[]
+
+[BCs]
+  [Periodic]
+    [all]
+      variable = 'disp_x'
+      auto_direction = 'z'
```

Comment:
Could you remind me why we only need `disp_x` here? Not `disp_y`?

Reply:
I went over the discussion earlier with Ritam. He experimented with different combinations of variables, only the `disp_x` gave the correct inverse pole figure. I simply kept what he had in the periodic BC in this test. 

Having both `disp_x` and `disp_y` changes the Euler angles values a bit, but the pole figures looks fairly similar to what I have in the documentation (generated with `disp_x` only).

Reply:
Thanks for your explanation.